virtualization/windowscontainers/quick-start/quick-start-windows-server.md

```diff
@@ -50,45 +70,66 @@ When PowerShell asks you whether to trust the package source 'DockerDefault', ty
 ```
 Restart-Computer -Force
 ```
+Please go to section 4 now, if you are happy with the Community Edition.
+
+You can find more information about Docker CE here [Docker Community Edition](https://www.docker.com/community-edition).
 
 > Tip: If you want to update Docker later:
 >  - Check the installed version with `Get-Package -Name Docker -ProviderName DockerMsftProvider`
 >  - Find the current version with `Find-Package -Name Docker -ProviderName DockerMsftProvider`
 >  - When you're ready, upgrade with `Install-Package -Name Docker -ProviderName DockerMsftProvider -Update -Force`, followed by `Start-Service Docker`
 
-## 2. Install Windows Updates
+## 2. Install Docker EE
```

Comment:
If these are the Insider instructions then this should be "Install Docker EE for Insider builds"